src/main/java/io/cdap/plugin/servicenow/source/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -49,9 +50,9 @@
   private static final String HEADER_NAME_TOTAL_COUNT = "X-Total-Count";
```

Comment:
looks like this is also defined in ServiceNowConstants. can we dedupe?

Reply:
Yes, removed the duplicates. [c5d6fbb](https://github.com/data-integrations/servicenow-plugins/pull/8/commits/c5d6fbb1b2d49b43e78aadd471064b92574c868e)